documentation/modules/upgrading/con-upgrade-custom-resources.adoc

```diff
@@ -0,0 +1,104 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='con-upgrade-custom-resources-{context}']
+
+= API versioning
+
+[role="_abstract"]
+Custom resources are created as instances of APIs added by CRDs to extend Kubernetes resources.
+CRDs are installed in a Kubernetes cluster to define the schemas used to instantiate and manage the custom resources.
+
+For release {ProductVersion}, Strimzi CRDs must specify `{ApiVersion}` as the API version and `{KafkaApiVersion}` as the API path for custom resources.
```

Comment:
I think this might confuse the timing. The new API version is really required only for Strimzi 0.23 - the next minor version.